semantic_conventions/trace/database.yaml

```diff
@@ -382,6 +382,45 @@ groups:
           The collection being accessed within the database stated in `db.name`.
         examples: [ 'customers', 'products' ]
 
+  - id: db.elasticsearch
+    prefix: db.elasticsearch
+    type: span
+    extends: db
+    brief: >
+      Call-level attributes for Elasticsearch
+    attributes:
+      - id: doc_id
+        type: string
+        requirement_level:
+          conditionally_required: when the request targets a specific document by id
+        tag: call-level-tech-specific-elasticsearch
+        brief: >
+          The document that the request targets.
+        examples: [ '123', '456' ]
+      - id: target
+        type: string
+        requirement_level:
+          conditionally_required: when a specific index or data stream is targeted by the request
+        tag: call-level-tech-specific
+        brief: >
+          The name of the data stream or index that is targeted.
+        examples: [ 'users' ]
+      - ref: url.full
+        requirement_level: required
+        examples: [ 'https://localhost:9200/index/_search?q=user.id:kimchy' ]
+      - ref: db.statement
+        requirement_level:
+          conditionally_required: when a search-type query is executed
+        brief: The request body for a [search-type query](https://www.elastic.co/guide/en/elasticsearch/reference/current/search.html), as a json string.
+        note: The value may be sanitized to exclude sensitive information.
+        examples: [ '"{\"name\":\"TestUser\",\"password\":\"REDACTED\"}"' ]
+      - ref: server.address
+        requirement_level: recommended
```

Comment:
nit, feel free to ignore: no need to specify the requirement level here, `recommended` is a default value